plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/embeddings/BackingFieldAccessors.kt

```diff
@@ -18,9 +18,13 @@ abstract class BackingFieldAccess(val field: FieldEmbedding) {
     ) {
         val fieldAccess = FieldAccess(receiver, field)
         val accPred = fieldAccess.getAccessPredicate()
-        ctx.addStatement(Stmt.Inhale(accPred))
+        if (field.inhalePolicy) {
```

Comment:
A policy should describe what happens (e.g. `INHALE_ON_ACCESS` may be a policy).  This seems like it's just a flag, so it would be better to name it after what it means or what it does.